docs/pages/guides/spend-permissions/api-reference/wallet-fetchpermissions.mdx

```diff
@@ -0,0 +1,45 @@
+## Coinbase Wallet API
+
+### `wallet_fetchPermissions`
+A utility API endpoint for recalling permissions previously granted for a specific user.
+Excludes any permissions that have been revoked or are expired.
+
+#### Schema
+```typescript
+type FetchPermissionsRequest = {
+  chainId: string; // uint256, hex formatted
+  account: string; // address
+  spender: string; // address
+  pageOptions?: {
+    pageSize number; // number of items requested, defaults to 50
+    cursor string; // identifier for where the page should start
+  }
+}
+
+type FetchPermissionsResult = {
+  permissions: FetchPermissionsResultItem[];
+  pageDescription: {
+    pageSize number; // number of items returned
+    nextCursor string; // identifier for where the next page should start
+  }
+}
+
+type PageDescription = 
```

Comment:
@amiecorso let's remove this or fill it out with the fields from `pageDescription` on lines 21-24